geometry_msgs/PoseStamped pose
float64 duration
bool enable
